UnityProject/Assets/Scripts/UI/Systems/AdminTools/AdminChatWindows.cs

```diff
@@ -75,7 +75,7 @@ public void ResetAll()
 			mentorPlayerChat.ClearLogs();
 			adminToAdminChat.ClearLogs();
 			playerPrayerWindow.ClearLogs();
-			adminLogWindow.ClearLogs();
+			//adminLogWindow.ClearLogs();
```

Comment:
intentional?

Reply:
there is no ClearLogs() function currently as entries get nuked whenever a page gets loaded, so this should probably be removed.